restli-server/src/main/java/com/linkedin/restli/server/resources/Jsr330Adapter.java

```diff
@@ -118,7 +118,7 @@ private void scan(final Collection<Class<?>> managedBeans)
   private void scanInjectableConstructors(Class<?> beanClazz)
   {
     int annotatedConstructors = 0;
-    for (Constructor<?> constructor : beanClazz.getConstructors())
+    for (Constructor<?> constructor : beanClazz.getDeclaredConstructors())
```

Comment:
Could you reasonably foresee this loosened behavior breaking any existing use cases? Could someone have written a resource with package-private or private constructors expecting them to not be used?

Reply:
Its highly unlikely that that would be the case, especially if the consumers are using any other JSR330 IOC framework.